Slack move to OAuth (#5241)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/job-notifications.md

### Prerequisites 
- You must be an administrator of the Slack workspace. 
- You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
- The integration only supports _public_ channels in the Slack workspace. 

### Set up the Slack integration

> 1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane. 

Reviewer: should this be "left sidebar"?

Author: Changed!

> 1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane. 

Author: ```suggestion
1. From the gear menu, select **Account settings** and then select **Integrations** from the left sidebar. 
```

1. Locate the **OAuth** section with the Slack application and click **Link**.

>    <Lightbox src="/img/docs/dbt-cloud/Link-your-Slack-Profile.png" width="75%" title="Link for the Slack app"/>

Reviewer: non-blocking: maybe as a fast-follow, update the screenshot so the Red Arrow does not cross any text

Author: Replaced it!

### Logged in to Slack

> If you are already logged in to Slack, the handover will only require allowing the app access. If you are a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.

Reviewer: ```suggestion
If you're already logged in to Slack, the handshake only requires allowing the app access. If you're a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.
```

   <Lightbox src="/img/docs/dbt-cloud/Allow-dbt-to-access-slack.png" width="75%" title="Allow dbt access to Slack"/>

## Logged out

> If you are logged out or the Slack app/website is closed, you will need to authenticate before completing the integraion.

Reviewer: ```suggestion
If you're logged out or the Slack app/website is closed, you must authenticate before completing the integration.
```

### Disable the Slack integration

1. From the gear menu, select **Account settings**. On the **Integrations** page, scroll to the **OAuth** section.

> 1. Click the trash can icon to the far right of the Slack integration, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.

Reviewer: ```suggestion
1. Click the trash can icon (on the far right of the Slack integration) and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.
```

### Disable the Slack integration

1. From the gear menu, select **Account settings**. On the **Integrations** page, scroll to the **OAuth** section.

> 1. Click the trash can icon to the right of the Slack integration, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.

Author: ```suggestion
1. Click the trash can icon to the far right of the Slack integration, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.
```
